builtin/providers/google/resource_compute_instance.go

```diff
@@ -58,6 +58,12 @@ func resourceComputeInstance() *schema.Resource {
 							ForceNew: true,
 						},
 
+						"nvme": &schema.Schema{
```

Comment:
Ah, I see. So you can't change the `scratch` field because that would cause a backwards-incompatible change with people that are already using the boolean. Since that field does have just the two values, I think leaving it be is just fine- we'll fix it when it becomes a problem. Note that the pattern I suggested is used plenty of places elsewhere in the provider, so this isn't going entirely against convention.

I can help you debug your test failures if you think it's a configuration problem. I'm also happy to run them myself to verify. However, I don't actually see any test changes in your PR. Mind adding one?